content/unit-testing/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Unit Testing"
+date: 2023-01-25T13:23:58-06:00
+draft: false
+weight: 100
+originalAuthor: <no value> # to be set by page creator
+originalAuthorGitHub: <no value> # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Learning Objectives
+
+By the end of this lesson, you should be comfortable with the following:
+
+1. Writing unit tests to verify the behavior of methods.
+1. Identifying proper behaviors for which a test should be written.
+1. The importance of testing in creating well-functioning code.
+1. Using test-driven development to define the behavior of code before writing it, and to ensure that code is properly tested.
+1. Using MSTest to create unit tests.
+
+## Key Terminology
```

Comment:
Please update the formatting for Key Terminology.

Reply:
Done!